deriva/config/catalog_config.py

```diff
@@ -0,0 +1,137 @@
+"""DERIVA catalog configuration utilities."""
+
+import logging
+from deriva.core import tag, urlquote
+from deriva.core.ermrest_model import Schema
+
+logger = logging.getLogger(__name__)
+
+
+def configure_baseline_ermrest_client(model, apply=True):
+    """Baseline configuration of `ERMrest_Client` table.
+
+    Set up `ERMrest_Client` table so that it has readable names and uses the display name of the user as the row
+    name.
+
+    :param model: an ermrest model object.
+    :param apply: if true, apply configuration changes before returning.
+    """
+    ermrest_client = model.schemas['public'].tables['ERMrest_Client']
+
+    # Make ermrest_client table visible.  If the GUID or member name is considered sensitive, then this needs to be
+    # changed.
+    ermrest_client.acls['select'] = ['*']
+
+    # Set table and row name.
+    ermrest_client.annotations.update({
+        tag.display: {'name': 'Users'},
```

Comment:
Isn't our normal convention for the display table name to be singular i.e. `User`?  It needs to work for both set and record views.

Reply:
You are right. Will change that.